servicex/templates/did-finder-rucio/deployment.yaml

```diff
@@ -35,8 +35,10 @@ spec:
           - name: RMQ_URI
             value: amqp://user:{{ .Values.rabbitmq.auth.password }}@{{ .Release.Name }}-rabbitmq:5672
     {{- end }}
-          - name: SITE
-            value: {{ if .Values.didFinder.rucio.site }} --site {{ .Values.didFinder.rucio.site }} {{ end }}
+          - name: RUCIO_LATITUDE
```

Comment:
Is the lat and long being passed in using the --site argument?  If so, how does the code differentiate between the two?

Reply:
this was messed up when redoing things. There is no argument. Rucio will simply take it from environment.